package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "0.5.8",
+  "version": "0.5.9",
```

Comment:
Another PR already changed this to 9. Please change yours to 10.

Reply:
Done